packages/botonic-core/src/types/index.d.ts

```diff
@@ -1,3 +1,13 @@
+export { Providers } from './constants'
+export { CoreBot } from './core-bot'
+export * from './debug'
+import { Inspector } from './debug'
+export * from './handoff'
+export { HubtypeService } from './hubtype-service'
+export { getString } from './i18n'
+export { Router } from './router'
+export * from './utils'
+
```

Comment:
Is it sorting the imports by file instead of object? Do we want it like this?

IMO it's easier to read them by object.

Reply:
yes, it is sorting them by file right now. Maybe we should discuss it and when we have the Pending PRs cleaner, apply the result of the discussion to all the packages

Reply:
Created a task to talk about this: https://linear.app/hubtype/issue/BOT-319/discussion-sort-imports-by-file-vs-sort-imports-by-object